Gems/Maestro/Code/Source/Components/EditorSequenceComponent.cpp

```diff
@@ -186,14 +199,24 @@ namespace Maestro
         Maestro::EditorSequenceComponentRequestBus::Handler::BusDisconnect();
         Maestro::SequenceComponentRequestBus::Handler::BusDisconnect();
 
+        AZ_Info("EditorSequenceComponent::Deactivate", "SequenceComponentRequestBus disconnected from %s", GetEntityId().ToString().c_str());
+
+        IMovieSystem* movieSystem = AZ::Interface<IMovieSystem>::Get();
+
+        // Remove this sequence from the game movie system.
+        if (m_sequence && movieSystem)
+        {
+            movieSystem->RemoveSequence(m_sequence.get());
+        }
+
         // disconnect from TickBus if we're connected (which would only happen if we deactivated during a pending property refresh)
         AZ::TickBus::Handler::BusDisconnect();
 
         EditorComponentBase::Deactivate();
     }
 
     ///////////////////////////////////////////////////////////////////////////////////////////////////////
-    void EditorSequenceComponent::AddEntityToAnimate(AZ::EntityId entityToAnimate)
+    bool EditorSequenceComponent::AddEntityToAnimate(AZ::EntityId entityToAnimate)
```

Comment:
maybe I'm mistaken, but haven't seen that return false in any implem, so why is the signature change needed ?